libs/auth-api-lib/migrations/20220823071133-rename-supportsdelegation-to-supportscustomdelegation.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.addColumn('client', 'supports_custom_delegation', {
+      type: Sequelize.BOOLEAN,
+      allowNull: false,
+      defaultValue: false,
+    })
+
+    await queryInterface.sequelize.query(`
+      UPDATE client SET supports_custom_delegation = supports_delegation
+    `)
+  },
+
+  async down(queryInterface, Sequelize) {
+    queryInterface.removeColumn('client', 'supports_custom_delegation')
```

Comment:
Should this be awaited?

```suggestion
    await queryInterface.removeColumn('client', 'supports_custom_delegation')
```

Reply:
Absolutely